Service/AddOn/ReadmeBuilder.php

```diff
@@ -393,29 +434,26 @@ protected function generateHtml() : string
          * @param string $tableTitle
          * @param ArrayCollection|array|Entity[] $entities
          * @param array $headerMap
+         * @param bool $groupedEntities
          */
-        $generateTableFromEntity = function (string $tableTitle, $entities, array $headerMap) use(&$readme) : void
+        $generateTableFromEntity = function (string $tableTitle, $entities, array $headerMap, $groupedEntities = false) use(&$readme) : void
         {
-            $block = '';
-
-            if (\count($entities))
+            $geneerateTableBodyFromEntity = function($entities, array $headerMap) use (&$block, &$readme)
             {
-                $block .= "<h2>{$tableTitle}</h2>";
                 $block .= '<table style="width:100%">';
-
                 // header row
                 $block .= '<thead><tr>';
-                foreach (\array_keys($headerMap) AS $header)
+                foreach (\array_keys($headerMap) as $header)
                 {
                     $block .= "<th>{$header}</th>";
                 }
                 $block .= '</tr></thead><tbody>';
 
                 // entity row
-                foreach ($entities AS $entity)
+                foreach ($entities as $entity)
                 {
                     $block .= '<tr>';
-                    foreach ($headerMap AS $header => $getter)
+                    foreach ($headerMap as $header => $getter)
```

Comment:
`as` must be all caps.

Reply:
That is not consistent with XF standards nor with the rest of your code.